packaging/opensuse-42.2/snapd.spec

```diff
@@ -149,6 +148,11 @@ go install -s -v -p 4 -x -tags withtestkeys github.com/snapcore/snapd/cmd/snapd
 %gobuild cmd/snapctl
 %gobuild cmd/snap-update-ns
 
+# This is ok because snap-seccomp only requires static linking when it runs from the core-snap via re-exec.
+sed -e "s/-Bstatic -lseccomp/-Bstatic/g" -i %{_builddir}/go/src/%{provider_prefix}/cmd/snap-seccomp/main.go
```

Comment:
We are also changing the affected file directly so which way is it going to be? I'd suggest actually using dynamic linking in the `main.go` there (common case) and patch that with sed for ubuntu (static linking, exception)